src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs

```diff
@@ -153,17 +169,35 @@ private static bool ContainsMultipleTypesInSpine(SyntaxNode node)
             }
         }
 
+        private static async Task<ImmutableArray<ISymbol>> GetMemberSymbolsAsync(Document document, SyntaxNode container, CancellationToken cancellationToken)
+        {
+            var semanticModel = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+            switch (container)
+            {
+                case TNamespaceDeclarationSyntax namespaceNode:
+                    var containerSymbol = (INamespaceSymbol)semanticModel.GetDeclaredSymbol(container);
+                    return containerSymbol.GetMembers().SelectAsArray(m => (ISymbol)m);
```

Comment:
cleaner to do `ImmutableArray.CastUp`

Reply:
it needs an ImmutableArray though, right?

Reply:
Yes.  But GetMemberss should be returning an immutablearray, no?

Reply:
No, it's IEnumerable :(